### ISSUE TITLE
Back to Search links to catalog.json

### ISSUE THREAD

**Issue by escowles** [pulibrary/plum#1098]
It should link to the regular catalog HTML view, not the JSON view.

**Comment by escowles** [comment]
@joycebcat Have you seen this problem recently?  I wasn't able to reproduce this now, by searching for items, creating new items, or editing existing items.  I'd appreciate it if you can help me reproduce this so we can debug and fix it.  If we can't reproduce this now, we can close it and assume some of the updates over the last month fixed it for us.

**Comment by joycebcat** [comment]
@escowles This still happens to me all the time.  I just logged in to Plum in Chrome.  I edited a scanned resource (Atlas of New Jersey), added a secondcollection to it and saved.  When it finished and the Back to Search button appeared, I clicked it and got the JSON view.  I still haven't ever had this NOT happen.  I'm usually using Safari.

**Comment by escowles** [comment]
@joycebcat Thanks for confirming it's still happening.  I tried a variety of things and found that it was happening in production and staging, but not when I run Plum locally in development mode.  The minimal set of steps to reproduce for me were:
* perform a search
* view a record
* go to the edit form
* go back to the record view
* reload the page

I've deployed a branch to staging and it's working for me there now.
